upstream/CVE-2019-11254.json

```diff
@@ -0,0 +1,89 @@
+{
+  "id": "CVE-2019-11254",
+  "modified": "2020-03-26T18:55:26Z",
+  "published": "2020-03-26T18:55:26Z",
+  "summary": "kube-apiserver Denial of Service vulnerability from malicious YAML payloads",
+  "details": "The Kubernetes API Server component in versions 1.1-1.14, and versions prior to 1.15.10, 1.16.7 and 1.17.3 allows an authorized user who sends malicious YAML payloads to cause the kube-apiserver to consume excessive CPU cycles while parsing YAML.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.1.0"
+            },
+            {
+              "fixed": "1.15.10"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.16.0"
+            },
+            {
+              "fixed": "1.16.7"
```

Comment:
issue says:
> v1.16.0-v1.16.7, resolved in 1.16.8 by https://github.com/kubernetes/kubernetes/pull/87639

v.1.16.7 contains this fix:
```bash
➜  git tag --contains a1e6da402568572f56e7605c0091f3e57009eb20 | sort -V
v1.16.7
```

Reply:
Updated `introduced` in https://github.com/aquasecurity/vuln-list-k8s/pull/74/commits/add0b31f22e20f8bd07abc4539cf74960bcde782
Updated `details` and added`reference` to this PR in https://github.com/aquasecurity/vuln-list-k8s/pull/74/commits/d0eabc232971b51b14fe4d7db6a82bca37fad336